### PR TITLE
Add push7 hass connection

### PR DESCRIPTION
**Pull request by L-I-Am**

### CODE REVIEW

File: d7a/alp/operations/responses.py [hunk 1]
```diff
@@ -25,36 +25,33 @@
...
     self.operand_class = Data
     super(ReturnFileData, self).__init__(*args, **kwargs)
-    self.try_parse_system_file()
+    self.try_parse_file(SystemFiles)
+    if custom_files_class is not None:
+      self.try_parse_file(custom_files_class)
 
-  def try_parse_system_file(self):
-    # when reading a known system files we store the parsed data and filename
+  def try_parse_file(self, file_class):
```
Comment (nielsheirman) [inline d7a/alp/operations/responses.py:42]: file_class -> files_class

File: d7a/alp/operations/responses.py [hunk 2]
```diff
@@ -25,36 +25,33 @@
...
     try:
-      systemfile_type = SystemFiles().files[SystemFileIds(self.operand.offset.id)]
+      file_type = file_class().files[file_class().enum_class(int(self.operand.offset.id))]
```
Comment (nielsheirman) [inline d7a/alp/operations/responses.py:44]: file_class does not need to be initiated to access enum_class as its a class variable
file_class.enum_class(....)

File: d7a/system_files/system_files.py [hunk 3]
```diff
@@ -33,6 +33,8 @@
 
 
 class SystemFiles:
+  enum_class = SystemFileIds
+
   files = {
     SystemFileIds.UID: UidFile(),
```
Comment (nielsheirman) [inline d7a/system_files/system_files.py:39]: The classes don't need to be instantiated here, as their main use will be to use the static method try_parse. In this case the class name is enough

Comment (L-I-Am) [reply d7a/system_files/system_files.py:39]: Not sure what you mean, these files are also used in the parse_hexstring where it does get used as a class. This is also not code that I changed.

File: d7a/alp/operations/responses.py (hunk 1 above)
Comment (L-I-Am) [reply d7a/alp/operations/responses.py]: changed it

File: d7a/alp/operations/responses.py (hunk 2 above)
Comment (L-I-Am) [reply d7a/alp/operations/responses.py]: changed it

File: d7a/system_files/system_files.py (hunk 3 above)
Comment (nielsheirman) [reply d7a/system_files/system_files.py:39]: nvm. The length field is actually accessed in some places, which is only defined on instance level

**Pull request merged**